src/main/java/org/folio/qm/service/population/impl/HoldingsLeaderMarcPopulationService.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.qm.service.population.impl;
+
+import java.util.Collections;
+
+import org.springframework.stereotype.Service;
+
+import org.folio.qm.domain.dto.MarcFormat;
+import org.folio.qm.service.population.LeaderMarcPopulationService;
+
+@Service
+public class HoldingsLeaderMarcPopulationService extends LeaderMarcPopulationService {
+
+  @Override
+  public boolean supportFormat(MarcFormat marcFormat) {
+    return marcFormat.equals(MarcFormat.HOLDINGS);
+  }
+
+  @Override
+  public String populateValues(String leader) {
```

Comment:
Should it be protected?

Reply:
No, it's supposed to be exposed and overload base method. Method name should be changed to overload base one.